### ISSUE TITLE
Link on npmjs.com is broken

### ISSUE THREAD

**Issue by manvir-singh** [oeuillot/node-omxplayer#3]
You seem to have renamed this repo at some point witch broke the link on npmjs.com.
Also the version at https://www.npmjs.com/package/omxplayer seems to be and older version that doesn't work 100% any more. Do you plan on keeping this project updated?

**Comment by oeuillot** [comment]
I fix it immediately !  (I don't understand what is going on !)
Regards

2016-10-13 3:24 GMT+02:00 Manvir notifications@github.com:

**Comment by oeuillot** [comment]
Hello,
Yes I have updated the version number to 0.0.2 !
I don't understand the problem

**Comment by oeuillot** [comment]
Is-it fixed ?

I am very interested by media-center  !!!!

**Comment by manvir-singh** [comment]
Yes! Everything works and there is now a new version on npmjs.com that works. Thanks for updating it.
Sense I already have this issue open I think semaphore should be removed and any kind of rate limiting should be done by the end user of the library.
